ui/app/AppLayouts/Profile/views/wallet/MainView.qml

```diff
@@ -15,8 +19,89 @@ Column {
 
     signal goToNetworksView()
 
+
+    StatusBaseText {
+        id: titleText
+        text: qsTr("Wallet")
+        font.weight: Font.Bold
+        font.pixelSize: 28
+        color: Theme.palette.directColor1
+    }
+
+    Item {
+        height: Style.current.bigPadding
+        width: parent.width
+    }
+
+    StatusSettingsLineButton {
```

Comment:
clicking on this open a sub menu and there is no way to go back, can we please add a back button or as per the design, maybe you are planning to do it in a subsequent task?

Reply:
Yes that will be in subsequent task, Here i am focusing on the layout without the data